stmt/stmt.go

```diff
@@ -115,6 +115,16 @@ type Labeled struct {
 	Stmt  Stmt
 }
 
+type Select struct {
+	Cases []CommCase
+}
+
+type CommCase struct {
+	Default bool
+	Stmt    Stmt // a recv- or send-stmt
+	Body    Stmt // always *BlockStmt
```

Comment:
By `*BlockStmt` you mean `*Block`?

Is anything stopping you from using the concrete type here?

Reply:
yes, copy-pasta.
I guess I could indeed just use `*Block`.